src/Command/LicenseReportCommand.php

```diff
@@ -0,0 +1,220 @@
+<?php
+/**
+ * @license
+ *
+ * Copyright (C) debricked AB
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code (usually found in the root of this application).
+ */
+
+namespace App\Command;
+
+use Debricked\Shared\API\API;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Contracts\HttpClient\Exception\ClientExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\RedirectionExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\ServerExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+class LicenseReportCommand extends Command
+{
+    use Style;
+
+    protected static $defaultName = 'debricked:license-report';
+
+    public const ARGUMENT_PROFILE = 'profile';
+    public const ARGUMENT_UPLOAD_ID = 'upload-id';
+    public const OPTION_FORMAT = 'format';
+    public const OPTION_OUTPUT_FILE = 'output';
+    public const OPTION_SNIPPETS = 'snippets';
+
+    /**
+     * @var HttpClientInterface
+     */
+    private $debrickedClient;
+
+    public function __construct(HttpClientInterface $debrickedClient, $name = null)
+    {
+        parent::__construct($name);
+
+        $this->debrickedClient = $debrickedClient;
+    }
+
+    protected function configure(): void
+    {
+        $findAndUploadCommand = FindAndUploadFilesCommand::getDefaultName();
+
+        $this
+            ->setDescription('Checks and prints status of given upload until it finish')
+            ->setHelp(
+                "Make sure to use run $findAndUploadCommand before so you have an upload to check."
+            )
+            ->addArgument(
+                FindAndUploadFilesCommand::ARGUMENT_USERNAME,
+                InputArgument::REQUIRED,
+                'Your Debricked username',
+                null
+            )
+            ->addArgument(
+                FindAndUploadFilesCommand::ARGUMENT_PASSWORD,
+                InputArgument::REQUIRED,
+                'Your Debricked password',
+                null
+            )
+            ->addArgument(
+                self::ARGUMENT_UPLOAD_ID,
+                InputArgument::REQUIRED,
+                "Upload id you got from running $findAndUploadCommand",
+                null
+            )
+            ->addArgument(
+                self::ARGUMENT_PROFILE,
+                InputArgument::REQUIRED,
+                'The license risk profile you wish to use for this scan',
```

Comment:
Can we list the possible values for this argument?

Reply:
Nice catch. Yes we can, and now we do :).